### ISSUE TITLE
Contain 'Skip to Main' link within a Landmark

### ISSUE THREAD

**Issue by cjstevens78** [localgovdrupal/localgov_theme#146]
During a recent accessibility review on Croydon via the AXE auditor - it was flagged that the 'Skip to Main' link should be contained within a landmark. As this change would benefit the localgov theme im adding the change here instead of within the Croydon theme.

**Comment by cjstevens78** [comment]
PR: https://github.com/localgovdrupal/localgov_theme/pull/147
